bitnami/mongodb/templates/replicaset/statefulset.yaml

```diff
@@ -450,7 +450,7 @@ spec:
           emptyDir: {}
         - name: certs-volume
           secret:
-            secretName: ca-clients
+            secretName: {{ default "ca-clients" .Values.tls.existingSecret }}
```

Comment:
I just noticed the default secret is hardcoded to "ca-clients", which is clearly wrong. We should use sth like:'
```
  name: {{ include "mongodb.fullname" . }}-metrics
```

Could you please fix it both here and the **secrets-ca.yaml** file (line 9)?

Reply:
@juan131 updated, does this look ok? Pulling it out into a secret seemed like the easiest solution